tests/helper_functions.py

```diff
@@ -154,8 +154,7 @@ def pyfunc_serve_and_score_model(
                                              declaring the scoring process to have failed.
     :param extra_args: A list of extra arguments to pass to the pyfunc scoring server command. For
                        example, passing ``extra_args=["--env-manager", "local"]`` will pass the
-                       ``--env-manager local`` flag to the scoring server to ensure that conda
-                       environment activation is skipped.
+                       ``--env-manager local`` flag to the scoring server to ensure that conda environment activation is skipped.
     """
     env = dict(os.environ)
```

Comment:
Can we add a = 1 (unused variable) above this line as a test?

pylint should raise this error: http://pylint-messages.wikidot.com/messages:e0602